### DISCUSSION TITLE
Why it's `.mbs` not just a `.json` file ?

### DISCUSSION THREAD

**Discussion by yharby** [ldproxy/ldproxy#1271] [General]
I have a very simple question: why does this file have the extension `.mbs` (Mapbox Style) instead of `.json` for simplicity?
- https://github.com/interactive-instruments/ldproxy/blob/master/demo/vineyards/api-resources/styles/vineyards/default.mbs

**Comment by azahnen** [comment]
Both extensions are supported in v4.x, `.mbs` is legacy, `.json` is encouraged. The demo folder in the ldproxy repository is actually not up-to-date, see https://github.com/ldproxy/demo instead.
